Detectors/GlobalTrackingWorkflow/tofworkflow/include/TOFWorkflow/TOFClusterizerSpec.h

```diff
@@ -18,7 +18,7 @@ namespace o2
 namespace tof
```

Comment:
this one also

Reply:
Also general comment: the GlobalTrackingWorkflow is in principle for cross-detector DPL deviced. I would keep the specs for those devices which require TOF data only, in the TOF/workflow directory, e.g. see ``O2/Detectors/ITSMFT/ITS/workflow``